src/sql-parser/src/parser.rs

```diff
@@ -757,35 +742,10 @@ impl Parser {
         debug!("parsing infix");
         let tok = self.next_token().unwrap(); // safe as EOF's precedence is the lowest
 
-        let regular_binary_operator = match tok {
+        let regular_binary_operator = match &tok {
+            Token::Op(s) => Some(s.as_str()),
             Token::Eq => Some("="),
-            Token::Neq => Some("<>"),
```

Comment:
Yay. I did the same thing in my rust sql smither because sql-parser's operator view was super strict--they should just be strings!